src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings

```diff
@@ -2662,3 +2662,143 @@ Permission_ImpersonateAnyLogin = Impersonate Any Login
 
 ServiceProviderNotSet = SetServiceProvider() was not called to establish the required service provider
 ServiceNotFound(string serviceName) = Service {0} was not found in the service provider
+
+############################################################################
+# Create Database
```

Comment:
A bunch of localized strings were missing from the DatabasePrototype code, so I copied over every Create Database string from SSMS just to be safe. Previously we wouldn't even get messages for exceptions because the localized strings weren't defined.